eea/facetednavigation/widgets/sorting/widget.py

```diff
@@ -109,9 +109,12 @@ def vocabulary(self, **kwargs):
         """ Return data vocabulary
         """
         vocab = self.portal_vocabulary()
-        sort_fields = self.listSortFields()
+        sort_fields = [x for x in self.listSortFields()]
+
         if not vocab:
-            return [field for field in sort_fields]
+            return sort_fields
+
+        vocab_fields = [(x[0], x[1], '') for x in vocab]
+        sort_field_ids = [x[0] for x in sort_fields]
 
-        vocab_fields = [field[0].replace('term.', '', 1) for field in vocab]
-        return [f for f in sort_fields if f[0] in vocab_fields]
+        return [f for f in vocab_fields if f[0] in sort_field_ids]
```

Comment:
Ops. You removed `field[0].replace('term.', '', 1)`. I think this is helpful when you use it with Products.ATVocabularyManager vocabularies. 

@tsimkins Can you fix this, please?